controllers/users.js

```diff
@@ -5,14 +5,23 @@ const UsersController = {
     res.render("users/new", {});
   },
 
-  Create: (req, res) => {
+  Create: async (req, res) => {
     const user = new User(req.body);
-    user.save((err) => {
-      if (err) {
-        throw err;
-      }
-      res.status(201).redirect("/posts");
-    });
+
+    const userExists = await User.exists({email: user.email})
+
+    if (userExists == false) {
+      user.save((err) => {
+        if (err) {
+          throw err;
+        }
+        res.status(201).redirect("/posts");
+      });
+    } else {
+      // should flash an error message back to the user
+      res.redirect("/users/new");
```

Comment:
we think this should redirect the existing user to sessions/new (the log in page).  This looks like it's sending the existing user to the new user page.